moler/cmd/at/genericat.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""
+Common part for all commands.
+
+AT commands specification:
+google for: 3gpp specification 27.007
+(always check against latest version of standard)
+"""
+
+__author__ = 'Grzegorz Latuszek'
+__copyright__ = 'Copyright (C) 2020, Nokia'
+__email__ = 'grzegorz.latuszek@nokia.com'
+
+import re
+import abc
+import six
+from moler.cmd.commandtextualgeneric import CommandTextualGeneric
+from moler.exceptions import CommandFailure
+from moler.exceptions import ParsingDone
+
+
+class AtCommandModeNotSupported(Exception):
```

Comment:
Why special exceptions for AT commands? We have CommandFailure as exception for all command failures in Moler.